Translations/font_tables.py

```diff
@@ -325,6 +325,70 @@ def get_font_map_latin_extended() -> Dict[str, bytes]:
     }
     return font
 
+#TODO: Add Greek Font
```

Comment:
Why the TODO?

Reply:
Oh sorry, chronic brain damage from watching Stanley Kubrick